### DISCUSSION TITLE
Possibility of scrypted nvr sessions that do not expire

### DISCUSSION THREAD

**Discussion by 50l3r** [koush/scrypted#1351] [Q&A] [answered]
I bought a minipc to be able to always have the web version of scrypted nvr open. That way I have a monitor that shows me all the cameras 24 hours a day.

The problem is that every day or so I have to log in again manually.

Is there any way for a session not to expire?

**Comment by koush** [comment, marked as answer]
Install as PWA for long term sessions.

**Comment by 50l3r** [comment]
Is there any other way to apply it to the web version?

I use a custom script that uses puppeter to rotate between certain web pages every so often. 

The pwa version also leaves the windows context menu on top:

![image](https://github.com/koush/scrypted/assets/5095592/5f0dd5b4-a48b-4083-8de0-0364acdeb17a)

**Comment by koush** [comment]
I see, I'll maybe add an option, but its not a trivial fix at the moment.

**Comment by 50l3r** [comment]
Much thanks @koush
